main.go

```diff
@@ -42,4 +42,23 @@ func main() {
 	}
 
 	btcLC.Status()
+
```

Comment:
We shouldn't do all logic testing in main. Let's test for those method. 
This must be good for in other PR if we already need this (Because we also move this code to another repo and change test config also)